ui-admin/src/study/participants/participantList/search/AdvancedSearchModal.test.tsx

```diff
@@ -1,10 +1,21 @@
 import React from 'react'
 import { mockStudyEnvContext } from 'test-utils/mocking-utils'
-import { findByText, fireEvent, getByText, render, screen } from '@testing-library/react'
+import {
+  findByText,
+  fireEvent,
+  getByText,
+  render,
+  screen
+} from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
 import AdvancedSearchModal from './AdvancedSearchModal'
 import { setupRouterTest } from '@juniper/ui-core'
 
+jest.mock('../../../../api/api', () => ({
```

Comment:
I don't think you need the ...
```suggestion
jest.mock('api/api', () => ({
```

Reply:
I finally figured out how to change intellij to never import relative 😄 I have no idea why someone would prefer this ugly `../../` chaining... Apologies for this being in almost all of my PRs up until now!